packages/flutter/lib/src/material/button_style.dart

```diff
@@ -435,3 +435,19 @@ class _LerpShapes implements MaterialStateProperty<OutlinedBorder?> {
     return ShapeBorder.lerp(resolvedA, resolvedB, t) as OutlinedBorder?;
   }
 }
+
+/// How the icon in any `button.icon` should be placed, where `button` could be:
+/// * [TextButton]
+/// * [ElevatedButton]
+/// * [OutlinedButton]
+///
+/// See also:
+///
+/// * [TextButton], [ElevatedButton], [OutlinedButton], the button widgets
```

Comment:
we probably don't need the See also section here since it just repeats the list just above.

Reply:
Okay, will remove it.